src/Sulu/Bundle/AdminBundle/Resources/js/components/Url/tests/Url.test.js

```diff
@@ -140,7 +140,7 @@ test('Call onChange callback when path was changed but not blurred', () => {
     expect(changeSpy).toBeCalledWith('https://sulu.at');
 });
 
-test('Call onChange callback with undefined when path was changed to invalid url but not blurred', () => {
+test('Call onChange callback when path was changed to invalid url but not blurred', () => {
```

Comment:
does this test make any sense as the url is I think from our case of view valid, or do I missunderstand and there is still a validation done?

Reply:
I just want to test, that the onChange callback is now be called with the actual url instead of `undefined` like it was before.